.github/workflows/test-webapp.yaml

```
@@ -44,12 +44,26 @@
+
+      # check for npm package blacklists pkgs across all packages
+      - uses: rainlanguage/github-chore/•github/actions/npm-blacklist@main
+
+      - uses: rainlanguage/github-chore/•github/actions/npm-blacklist@main
+        with:
+          working-directory: packages/orderbook
+
+      - uses: rainlanguage/github-chore/•github/actions/npm-blacklist@main
+        with:
+          working-directory: packages/ui-components
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)